ServerSide/yoty/Data/Entities/ParticipancyEntity.cs

```diff
@@ -5,18 +5,18 @@ namespace YOTY.Service.Data.Entities
 {
     public class ParticipancyEntity
     {
-        public int Id { get; set; }
+        public int NumOfUnits { get; set; }
+
+        //-----------------------------
+        //Relationships
+        [Required]
+        public BuyerEntity Buyer { get; set; }
```

Comment:
I think you explained it to me but I still don't sure why we we need here the buyer and bid and just their ids